tools/diann/diann.xml

```diff
@@ -0,0 +1,486 @@
+<tool id="diann" name="DIA-NN" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@">
+    <description>is a software for DIA/SWATH data processing</description>
+    <macros>
+        <token name="@TOOL_VERSION@">1.8.1</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <requirements>
+        <container type="docker">biocontainers/diann:v@TOOL_VERSION@_cv1</container>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        mkdir ./input_data &&
+        mkdir ./tmp &&
+
+        ## generate string like "--f file1 --f file2 ..." for each input in $input.f, which are comma separated
+        #set infiles_str = ''
+        #for $infile in $input.f
+            ## if brukertdf.d.tar, extract to ./input_data and append "./input_data/" to infiles_str
+            #if $infile.is_of_type("brukertdf.d.tar"):
+                tar -xf '$infile' -C ./input_data &&
+                #set $infiles_str += '--f ./input_data/' + $infile.name[:-4] + ' '
+            #else
+                #set $infiles_str += '--f ' + $infile.name + ' '
+            #end if
+        #end for
+
+        diann-@TOOL_VERSION@
```

Comment:
I would need to check the container when I'm home, but are you sure the binary is called after the version number?

Reply:
no, just found out it's not the case.........

Reply:
the command in the docker is just `diann` and not `diann-1.8.1`